src/Option.js

```diff
@@ -1,17 +1,24 @@
 import React, { Component, PropTypes } from 'react';
 import { PREFIX } from './index';
-import { pureRender } from './utils';
+import { pureRender, getOptionValue } from './utils';
+
+const defaultRenderOption = (option) => <span>{option.label}</span>;
 
 class Option extends Component {
 
   static propTypes = {
     index: PropTypes.number,
-    label: PropTypes.string,
-    value: PropTypes.string,
+    option: PropTypes.shape({
+      label: PropTypes.string,
+      value: PropTypes.string,
+    }),
+
```

Comment:
ที่ทำแบบนี้เพราะว่าเผื่อกรณีจะ renderOption โดยใช้ parameter มากกว่า value, label ปกติน่ะ เช่นสมมติ

const options = [
  { value: 'sven', label: 'Sven', trait: 'Strength' },
  { value: 'drowranger', label: 'Drow Ranger', trait: 'agility' },
  { value: 'naix', label: 'Nyx', trait: 'inteligent' },
];

แล้วเราโยนทั้ง option ไปที่ renderOption (option) 

ใน render เราสามารถเพิ่มเงื่อนไขได้เช่น

render() {
  const option = this.props.option;
  switch (option.trait) {
    case 'Strength': return this.renderStrentghHeroOption(option);
    case 'Agility': return this.renderAgilityHeroOption(option);
  }
  

}